packages/phishing/src/additions.spec.ts

```diff
@@ -7,7 +7,14 @@ import { decodeAddress } from '@polkadot/util-crypto';
 
 const addresses = JSON.parse(fs.readFileSync('address.json', 'utf-8')) as Record<string, string[]>;
 const allowed = JSON.parse(fs.readFileSync('known.json', 'utf-8')) as Record<string, string[]>;
-const all = JSON.parse(fs.readFileSync('all.json', 'utf8')) as { deny: string[] };
+const all = JSON.parse(fs.readFileSync('all.json', 'utf8')) as { allow: string[]; deny: string[] };
+
+const TOP_LEVEL = [
+  'pages.dev',
+  'servehttp.com',
+  'vercel.app',
+  'web.app'
+];
```

Comment:
@dubstard Any others that you can think of?

Reply:
perhaps `plesk.page` and also `webflow.io`
(unfortunately seeing a lot of scams linked to both)